src/mito2/src/error.rs

```diff
@@ -684,6 +690,7 @@ impl ErrorExt for Error {
             BiError { .. } => StatusCode::Internal,
             EncodeMemtable { .. } | ReadDataPart { .. } => StatusCode::Internal,
             ChecksumMismatch { .. } => StatusCode::Unexpected,
+            RegionStopped { .. } => StatusCode::Internal,
```

Comment:
Looks like `StatusCode::RegionNotReady` would be better

https://greptimedb.rs/common_error/status_code/enum.StatusCode.html#variant.RegionNotReady